src/kernels/jupyter/launcher/jupyterConnection.node.ts

```diff
@@ -175,7 +173,7 @@ export class JupyterConnectionWaiter implements IDisposable {
     }
 
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
The lint rule comment can probably be removed

Reply:
Oops, missed that, sory, will submit another PR